tests/phpunit/CommonExtractorTest.php

```diff
@@ -4,35 +4,41 @@
 
 namespace Keboola\DbExtractor\Tests;
 
-use Keboola\Csv\CsvFile;
+use PDO;
+use Monolog\Logger;
+use Keboola\Csv\CsvReader;
 use Keboola\DbExtractor\Application;
 use Keboola\DbExtractor\Exception\ApplicationException;
 use Keboola\DbExtractor\Exception\UserException;
 use Keboola\DbExtractorConfig\Exception\UserException as ConfigUserException;
-use Keboola\DbExtractorLogger\Logger;
 use Monolog\Handler\TestHandler;
+use PHPUnit\Framework\Assert;
 use Symfony\Component\Filesystem\Filesystem;
 use Symfony\Component\Finder\Finder;
 use Keboola\DbExtractor\Test\ExtractorTest;
 use Keboola\DbExtractor\Test\DataLoader;
+use Symfony\Component\Process\Process;
 
 class CommonExtractorTest extends ExtractorTest
 {
     public const DRIVER = 'common';
 
-    /** @var string */
-    protected $appName = 'ex-db-common';
+    protected string $appName = 'ex-db-common';
 
-    /**
-     * @var  \PDO
-     */
-    private $db;
+    private PDO $db;
 
-    public function setUp(): void
+    protected function setUp(): void
     {
         $this->initDatabase();
     }
 
+    protected function tearDown(): void
+    {
+        # Close SSH tunnel if created
+        $process = new Process(['sh', '-c', 'pgrep ssh | xargs -r kill']);
```

Comment:
stačí jednodušeji `pkill ssh`

Reply:
IMHO nestaci, `pkill` skonci chybou, ... ak ziadny taky proces neexistuje:
![image](https://user-images.githubusercontent.com/19371734/80084760-9663e700-8557-11ea-8f8b-5572342a04a5.png)

Samozrejme da sa to vyriesit v PHP, ale podla mna to je jedno.

Reply:
aha to je zvláštní, ok